pkg/operation/botanist/component/vpnshoot/vpnshoot.go

```diff
@@ -297,15 +298,38 @@ func (v *vpnShoot) computeResourcesData(secretCAVPN *corev1.Secret, secretsVPNSh
 		podSecurityPolicy *policyv1beta1.PodSecurityPolicy
 		clusterRolePSP    *rbacv1.ClusterRole
 		roleBindingPSP    *rbacv1.RoleBinding
+
+		labels = map[string]string{
+			v1beta1constants.GardenRole:     v1beta1constants.GardenRoleSystemComponent,
+			v1beta1constants.LabelApp:       LabelValue,
+			managedresources.LabelKeyOrigin: managedresources.LabelValueGardener,
+		}
+		template = v.podTemplate(serviceAccount, secretsVPNShoot, secretCA, secretTLSAuth, secretDH)
+
+		networkPolicyFromSeed = &networkingv1.NetworkPolicy{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "gardener.cloud--allow-from-seed",
```

Comment:
Can we keep this in a separate variable instead of hard-coding it here?

Reply:
What for, I think that's the only place where this name is used, isn't it?